### PR TITLE
FFTF-582: Fix hint text on irrigated-land page

### PR DESCRIPTION
**Pull request by diya2214**

### CODE REVIEW

File: app/routes/irrigated-land.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ function createModel (irrigatedLandCurrent, irrigatedLandTarget, errorMessageLis
         text: 'ha'
       },
       hint: {
-        text: 'Enter figure in hectares'
+        html: '<font color="black"> Hectares </font><br/> Enter figure in hectares, for example 543.5'
```
Comment (ogbk) [inline app/routes/irrigated-land.js:28]: <span class="govuk-label govuk-!-font-weight-bold"> Hectares </span>Enter figure in hectares, for example 543.5'

File: app/routes/irrigated-land.js [hunk 2]
```diff
@@ -40,7 +40,7 @@ function createModel (irrigatedLandCurrent, irrigatedLandTarget, errorMessageLis
       id: 'irrigatedLandTarget',
       name: 'irrigatedLandTarget',
       hint: {
-        text: 'Enter figure in hectares'
+        html: '<font color="black"> Hectares </font><br/> Enter figure in hectares, for example 543.5'
```
Comment (ogbk) [inline app/routes/irrigated-land.js:43]: <span class="govuk-label govuk-!-font-weight-bold"> Hectares </span>Enter figure in hectares, for example 543.5'

File: app/routes/irrigated-land.js (hunk 1 above)
Comment (ogbk) [inline app/routes/irrigated-land.js:28]: html: '<span class="govuk-label govuk-!-font-weight-bold"> Hectares </span>Enter figure in hectares, for example 543.5'

Comment (ogbk) [inline app/routes/irrigated-land.js:28]: '<span class="govuk-label govuk-!-font-weight-bold"> Hectares </span>Enter figure in hectares, for example 543.5'

Comment (ogbk) [inline app/routes/irrigated-land.js:28]: <span class="govuk-label govuk-!-font-weight-bold">

Comment (ogbk) [inline app/routes/irrigated-land.js:28]: `html: '<span class="govuk-label govuk-!-font-weight-bold"> Hectares </span>Enter figure in hectares, for example 543.5'`

Comment (ogbk) [inline app/routes/irrigated-land.js:28]: `<span class="govuk-label govuk-!-font-weight-bold"> Hectares </span>`

File: app/routes/irrigated-land.js (hunk 2 above)
Comment (ogbk) [inline app/routes/irrigated-land.js:43]: `<span class="govuk-label govuk-!-font-weight-bold"> Hectares </span>`

Comment (ogbk) [reply app/routes/irrigated-land.js:43]: I think this is better than `<font color = "black">` - we should always use govuk stylesheets, however `govuk-!-font-weight-bold` may or may not be want the content design wants - please check it out locally & decide

Comment (diya2214) [reply app/routes/irrigated-land.js]: done

**Review by ogbk — APPROVED** [review]

**Pull request merged**